cmd/process.go

```diff
@@ -49,16 +49,14 @@ func Start(a *AppState) *cobra.Command {
 
 			flushOnly, err := cmd.Flags().GetBool(flagFlushOnlyMode)
 			if err != nil {
-				logger.Error("Invalid flush only flag", "error", err)
-				os.Exit(1)
+				return fmt.Errorf("invalid flush only flag error=%e", err)
```

Comment:
when using `fmt.Errorf` to create a new error, if you are passing in an existing error as an argument to the format specifier string you can use the `%w` verb to retain the original error. essentially you create a new error that implements the `Unwrap` method that allows you to retrieve the original error, which can be useful for debugging and handling certain errors since you can track the entire chain of errors as you wrap new information around an existing error. see: https://pkg.go.dev/fmt#Errorf

since returning this error essentially leads to the program terminating it isn't as important since we are unlikely to need to perform some action based off the error type, but i figured i would point this out!